sdks/java/io/hadoop-input-format/jdk1.8-hifio-tests/src/test/resources/kubernetes/elasticsearch/es-client-node-rc.yaml

```diff
@@ -0,0 +1,86 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Deployment script for creating Elasticsearch nodes for client usage.
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: es-client
+  labels:
+    component: elasticsearch
+    role: client
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+        role: client
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
```

Comment:
Added comment :"  # Elasticsearch uses a hybrid mmapfs / niofs directory by default to store its indices. 
      # The default operating system limits on mmap counts is likely to be too low, which may result
      # in out of memory exceptions. Therefore, the need to increase virtual memory 
      # vm.max_map_count for large amount of data"